.github/workflows/ci.yml

```
@@ -91,14 +91,15 @@ jobs:
       - run: pip install coverage[toml]
 
       - name: download coverage data
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        uses: actions/download-artifact@7a1cd3216ca9260cd8022db641d960b1db4d1be4 # v4.0.0
         with:
-          name: coverage-data
+          path: all-artifacts/
 
       - name: combine coverage data
         id: combinecoverage
         run: |
           set +e
+          cp all-artifacts/coverage-data-*/.coverage.* .
           python -m coverage combine
```

This is a tiny thing, but I _think_ you can do this: 

```suggestion
          set +e
          python -m coverage combine all-coverage/
```

---

thanks, that's better. The argument needs to be `all-artifacts/coverage-data-*` though (download-artifacts creates extra directories when used like this).